master/internal/resourcemanagers/kubernetes_resource_manager.go

```diff
@@ -55,10 +68,19 @@ func (k *kubernetesResourceManager) Receive(ctx *actor.Context) error {
 	case actor.PreStart:
 		actors.NotifyAfter(ctx, actionCoolDown, schedulerTick{})
 
-	case sproto.SetPods:
-		check.Panic(check.True(k.agent == nil, "should only set pods once"))
+		podsActor := kubernetes.Initialize(
```

Comment:
non-blocking clarification: is this pods actor or a resource manager being created?

Reply:
This is to create the pods actor when "Prestarting" the k8s resource manager.